API_CHANGELOG.md

```diff
@@ -57,4 +57,4 @@ are ignored.  PR #271
 
 * Removed all matrix libraries.  Use the `Connections` class instead.  PR #169
 
-* Removed `void SpatialPooler::stripUnlearnedColumns()` as unused and not useful (did not effectively remove any columns). https://github.com/htm-community/nupic.cpp/pull/286 
+* Removed `void SpatialPooler::stripUnlearnedColumns()` as unused and not useful (did not effectively remove any columns). PR #286 
```

Comment:
@ctrl-z-9000-times just a mention, next time I'd prefer the full URL format. To a user reading the document in browser, it both shows as "#286", but the `#286` breaks if you copy the code to a repo, or what not.

Reply:
If that is the case then you should update all of the cross references in the API_Changelog.  I changed it to a short link for consistency with the rest of the file.

Reply:
> update all of the cross references in the API_Changelog

well, I probably don;t care enough to do that now. Was just saying for next time...